### PR TITLE
feat(verify): only display verified comments

### PR DESCRIPTION
**Pull request by sdodson99**
# Description

Require manual verification of comments, as discussed in #12.

# Changes

- Filter comments that are not verified (`verified === false`)
- Update integration tests to add advanced querying support
- Document required Firestore security rules for comment verification to work
- Document steps for verifying comments

### CODE REVIEW

File: package/test/integration/setup/firebase.ts [hunk 1]
```diff
@@ -11,12 +11,17 @@ vi.mock('firebase/firestore', async (importOriginal) => {
...
+    query: vi.fn(),
+    where: vi.fn(),
   };
 });
 
 export const mockFirebaseApp = {} as FirebaseApp;
 
 const mockFirestoreCollection = collection as Mock;
+const mockFirestoreQuery = query as Mock;
+const mockFirestoreWhere = where as Mock;
```
Comment (edodso2) [inline package/test/integration/setup/firebase.ts:23]: Do we need to reset these mocks as well in the `afterEach`?

**Review by edodso2 — COMMENTED** [review]
Super clean! The firebase mocks are super trick, had to go over that a few times to fully comprehend what was happening there 😅

File: package/test/integration/setup/firebase.ts (hunk 1 above)
Comment (sdodson99) [reply package/test/integration/setup/firebase.ts:23]: good catch, updated 👍

**Comment by sdodson99** [comment]
> Super clean! The firebase mocks are super trick, had to go over that a few times to fully comprehend what was happening there 😅

It is super disconnected 😅 The Firestore query interface is weird too, like having to pass the comparison to `where()` as a string

**Review by edodso2 — APPROVED** [review]
🚀

**Pull request merged**